galgebra/printer.py

```diff
@@ -1083,30 +1083,19 @@ def LatexFormat(Fmode=True, Dmode=True, ipy=False):
     GaLatexPrinter.redirect()
     return
 
-prog_str = ''
 off_mode = False
 
 
 def Get_Program(off=False):
-    global prog_str, off_mode
+    global off_mode
     off_mode = off
```

Comment:
This function is now pretty useless - I'll deprecate it in a follow-up I think.

Reply:
@abrombo used for displaying the program and the output in the same LaTeX file

Reply:
That's what `Print_Function` is for. All this function does is say "make Print_Function do nothing when I call it".

Reply:
Oh

Reply:
I always confuse these two